lib/money-rails/active_record/monetizable.rb

```diff
@@ -85,10 +85,10 @@ def monetize(*fields)
             #
             # To disable validation entirely, use :disable_validation, E.g:
             #   monetize :price_in_a_range_cents, :disable_validation => true
-            if validation_enabled = MoneyRails.include_validations && !options[:disable_validation]
+            if (validation_enabled = MoneyRails.include_validations && !options[:disable_validation])
```

Comment:
Yes. This one doesn't generate a Ruby warning if I recall correctly, but I thought I may as well fix all the RuboCop lint offenses in one go, rather than in dribs and drabs. If you don't want this as part of the commit that's fine though.